sqlparser_bench/benches/sqlparser_bench.rs

```diff
@@ -42,6 +42,46 @@ fn basic_queries(c: &mut Criterion) {
     group.bench_function("sqlparser::with_select", |b| {
         b.iter(|| Parser::parse_sql(&dialect, with_query));
     });
+
```

Comment:
For `large_statement`, making separated test would make differentiating potential(future) regression easier.
It's a suggestion(fine to leave it as it is).

Reply:
Not sure I understood your comment, sorry. I added tests for parsing large statements in tests/sqlparser_common.rs. Do you think we should test something else?

Reply:
I thinks it would be better to add as separated test.